src/Simulation.cpp

```diff
@@ -58,6 +58,12 @@ void Simulation::run(int steps) {
     Vector3d f_net{};
     Vector3d w_net{};
 
+    bool drogue_deploy;
+    bool main_deploy;
+
+    double Cd;
+    double A_ref;
+
```

Comment:
These variables should be declared where they are used. (r_vect and friends should be too, but thats outside the scope of this pr)